corehq/toggles.py

```diff
@@ -1472,7 +1472,11 @@ def _commtrackify(domain_name, toggle_is_enabled):
 
 FILTERED_BULK_USER_DOWNLOAD = StaticToggle(
     'filtered_bulk_user_download',
-    "Bulk mobile worker management features: filtered download, bulk delete, and bulk lookup users.",
+    """
+        Bulk user management features:
+        For mobile users, enables filtered download page, bulk deletion page, and bulk lookup page.
+        For web users, enables filtered download page.
+    """,
```

Comment:
so we are extending this new view to this old feature flag as well?

So the form for them would show only one domain as the option always, unless they have mirrored domains?

Reply:
yep, exactly